docs/mobile-apps/features/virtual-usb.md

```diff
@@ -164,7 +164,7 @@ Method 1 is recommended for the following reasons:
   11:13:12.347 INFO com.saucelabs.vusb.client.server.usbmuxd.SocketMover - This will require administrator privileges!
   ```
 
-  This prepares the usbmuxd socket (`/var/usbmuxd`) so that developer tools like Xcode can interact with the remote device just like they interact with a local device. You will need to have administrator permissions to replace `/var/usbmuxd` on your computer and [disable Systems Integrity Protection (SIP) in macOS](https://developer.apple.com/documentation/security/disabling_and_enabling_system_integrity_protection).
+  This prepares the usbmuxd socket (`/var/usbmuxd`) so that developer tools like Xcode can interact with the remote device just like they interact with a local device. You will need to have administrator permissions to replace `/var/usbmuxd` on your computer. After you did so please restart XCode to be able to interact with the devicec.
```

Comment:
@aluedeke 
We might need to be more clear on this, does the customer need to do this, or will the jar do this

Reply:
Something like: After the connection was established by the vUSB client, please restart all processes which are supposed to interact with the connected device (e.g. XCode)

Reply:
That's better, will you then also remove this / rephrase this

> You will need to have administrator permissions to replace `/var/usbmuxd` on your computer.